rollback_on_fail.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+rollback () {
+    wget https://github.com/Jellyfish-AI/jf_agent/archive/refs/tags/stable.tar.gz
+    tar -zxvf stable.tar.gz --directory ./
+    python jf_agent-stable/jf_agent/main.py
+}
+
+timeout --preserve-status 10h python jf_agent/main.py
+EXITCODE=$?
+# timeout will kill 143, config exception sys.exit's with 1, 
+# exceptions throw various; only a completed run exits with 0
+if [[ $EXITCODE -ne 0 ]]; then rollback; fi
```

Comment:
There's no way this would great an infinite loop, right? We're gonna run the initial run with `timeout`, and then run `rollback` only if it crashes (or timesout)? But we _won't_ run it again if rollback crashes again

Reply:
Yeah, sorry I had to write it out just to be sure